python/lsst/sims/catUtils/utils/testUtils.py

```diff
@@ -16,13 +23,296 @@ def calcADUwrapper(sedName=None, magNorm=None, redshift=None, internalAv=None, i
         if internalAv != 0.0 and internalRv != 0.0:
             a_int, b_int = sed.setupCCMab()
             sed.addCCMDust(a_int, b_int, A_v=internalAv, R_v=internalRv)
-    
+
     if redshift is not None and redshift != 0.0:
         sed.redshiftSED(redshift, dimming=True)
-    
+
     a_int, b_int = sed.setupCCMab()
     sed.addCCMDust(a_int, b_int, A_v=galacticAv, R_v=galacticRv)
-    
+
     adu = sed.calcADU(bandpass, photParams=PhotometricParameters())
-    
+
     return adu
+
+
+def makePhoSimTestDB(filename='PhoSimTestDatabase.db', size=1000, seedVal=32, radius=0.1,
+                     displacedRA=None, displacedDec=None,
+                     bandpass='r', m5=None, seeing=None, **kwargs):
+    """
+    Make a test database to storing cartoon information for the test phoSim input
+    catalog to use.
+
+    The method will return an ObservationMetaData object guaranteed to encompass the
+    objects in this database.
+
+    @param [in] filename is a string indicating the name of the DB file to be created
+
+    @param [in] size is the number of objects int he database
+
+    @param [in] seedVal is the seed passed to the random number generator
+
+    @param [in] radius is the radius (in degrees) of the field of view to be returned
+
+    @param [in] bandpass is the bandpas(es) of the observation to be passed to
+    ObservationMetaData (optional)
+
+    @param [in] m5 is the m5 value(s) to be passed to ObservationMetaData
+    (optional)
+
+    @param [in] seeing is the seeing value(s) in arcseconds to be passed to
+    ObservationMetaData (optional)
+
+    @param [in] displacedRA/Dec are numpy arrays that indicate where (in relation to the center
```

Comment:
It might be possible to clarify the meaning of this being a deviation from the center rather than the absolute position of the displaced radius if we use something like `deltaRA`. ("in relation to the center" helps, but might be hard to catch)